kano_feedback/DataSender.py

```diff
@@ -521,8 +579,12 @@ def send_question_response(answers, interactive=True, tags=['os', 'feedback-widg
     The answers will be all packed into a payload object and sent in one single network transaction.
     '''
 
-    ok_msg_title = _('Thank you')
-    ok_msg_body = _('We will use your feedback to improve your experience')
+    from kano_world.functions import get_email, get_mixed_username
+
+    ok_msg_title = _('Thank you')  # noqa: F821
```

Comment:
What does `noqa F821` mean?

Reply:
These are for Flake8 to ignore "function not defined" errors caused because `_()` gets added to the builtins. Ideally we wouldn't need this but any form of code analysis is better than what we currently have.